docker-compose.yaml

```diff
@@ -1,4 +1,6 @@
-﻿services:
+﻿name: lexbox
```

Comment:
the same thing `COMPOSE_PROJECT_NAME=lexbox` was doing, it's just better in the file rather than an ENV var.  Here's the doc ref:  https://docs.docker.com/compose/compose-file/04-version-and-name/#name-top-level-element